polars/polars-lazy/polars-plan/src/dsl/mod.rs

```diff
@@ -1493,6 +1493,25 @@ impl Expr {
         })
     }
 
+    #[cfg(feature = "cutqcut")]
+    pub fn nqcut(
```

Comment:
I am not happy with this name. I understand `qcut` is entrenched in existing software, so here we can deviate from our style conventions.

Could we maybe name this `qcut_spaced` or `qcut_n`?

Reply:
`qcut_uniform`?

Reply:
I like `qcut_uniform`. It's longer but I think it's much clearer and it's on the rust side where I see length as less of an issue since it's not being used for very "malleable" interactive work.

Reply:
Let's go for `qcut_uniform` then. :)